### ISSUE TITLE
Add the ability to set default colors for night and day for each UI component.

### ISSUE THREAD

**Issue by jlaws** [Draveness/DKNightVersion#19]
It would be great to be able to set defaults for global UIKit components so you don't need to specify the night and day color for every view.  Are you working on this?  If not I may have time to send out a PR.

**Comment by Draveness** [comment]
Yeah, I'm not. And I am very appreciate it if you could open a PR

**Comment by hyj223** [comment]
@jlaws 
 Have you done this work or not? I also consider it recently~

**Comment by jlaws** [comment]
I need permission from my company to contribute to any open source projects, so I haven't started yet.

**Comment by Draveness** [comment]
This issue are not on the schedule until complete 2.0 milestone
